### PR TITLE
Change word count to just say 'XXX words'.

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: explainer-server/app/views/explainEditor.scala.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
             </div>
             <div class="top-toolbar__item">
                 <p class="top-toolbar__item-inner word-count">
-                    word count: <span class="word-count__number"></span>
+                    <span class="word-count__number"></span> word(s)
```
Comment (shaundillon) [inline explainer-server/app/views/explainEditor.scala.html]: Needs a space

**Comment by philmcmahon** [comment]
I think we should add the logic so that it says word/words as composer does this

**Comment by shtukas** [comment]
screen shot 2016-08-31 at 12 03 59

screen shot 2016-08-31 at 12 04 13

File: explainer-server/public/javascripts/explain-editor-plain-js.js [hunk 2]
```diff
@@ -8,7 +8,9 @@ function getBodyWordCount(){
 }
 const maxWordCount = 150;
 function updateWordCountDisplay() {
-    $(".word-count__number").text(getBodyWordCount())
+    var count = getBodyWordCount()
+    var sentence = getBodyWordCount()>1 ? getBodyWordCount()+" words" : getBodyWordCount()+" word"
```
Comment (philmcmahon) [inline explainer-server/public/javascripts/explain-editor-plain-js.js]: Use the `count` variable rather than calling the same function 3 times

Comment (philmcmahon) [inline explainer-server/public/javascripts/explain-editor-plain-js.js]: Also, what happens when there are 0 words?

**Comment by shtukas** [comment]
> Use the count variable

Sorry I was meant to do that and forgot (starving, brain not working ^^)

**Comment by shaundillon** [comment]
👍

**Pull request merged**